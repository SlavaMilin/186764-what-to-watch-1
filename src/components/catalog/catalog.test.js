import React from 'react';
import renderer from 'react-test-renderer';
import Catalog from "./catalog";

it(`render catalog`, () => {
  const tree = renderer.create(<Catalog movies={[`first`]} genres={[`first`]} goToMovie={() => {}}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
