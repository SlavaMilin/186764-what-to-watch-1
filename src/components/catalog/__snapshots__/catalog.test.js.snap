// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render catalog 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>
  <ul
    className="catalog__genres-list"
  >
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        first
      </a>
    </li>
  </ul>
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
    >
      <button
        className="small-movie-card__play-btn"
        type="button"
      >
        Play
      </button>
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          first
        </a>
      </h3>
    </article>
  </div>
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>
</section>
`;
